portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,275 @@
+            CLICK ME
+          </button>
+        </div>
+        <h1>
+          Mariana Torres Torres
+        </h1>
+      </div>
+      <h1 id="typed"></h1>
+    </div>
+    <div id="about">
+      <div class="column">
+        <h1>
+          About Me
+        </h1>
+        <p>
+          Hey, I'm Mariana! I'm a second year Digital Arts and Sciences
+          Engineering student at the University of Florida. I was born and
+          raised in Caracas, Venezuela, but now I am based in Gainesville,
+          Florida. When I'm not coding, I like to draw, paint, cook, play
+          videogames, and read.
+        </p>
+        <p>
+          *I swear there's more to say about me, I'll just write it later lol*
+        </p>
+        <p>
+          Here's some of the languages and frameworks that I use:
+        </p>
+        <div class="about-list">
+          <ul>
+            <li>
+              <span><i class="fas fa-plus"></i></span>JavaScript
+            </li>
+            <li>
+              <span><i class="fas fa-plus"></i></span>C++
+            </li>
+            <li>
+              <span><i class="fas fa-plus"></i></span>Java
+            </li>
+            <li>
+              <span><i class="fas fa-plus"></i></span>HTML & CSS
+            </li>
+            <li>
+              <span><i class="fas fa-plus"></i></span>Processing
+            </li>
+            <li>
+              <span><i class="fas fa-plus"></i></span>Python
+            </li>
+          </ul>
+        </div>
+        <div class="about-list">
+          <ul>
+            <li>
+              <span><i class="fas fa-plus"></i></span>React & React Native
+            </li>
+            <li>
+              <span><i class="fas fa-plus"></i></span>NodeJS
+            </li>
+            <li>
+              <span><i class="fas fa-plus"></i></span>Apollo & GraphQL
+            </li>
+            <li>
+              <span><i class="fas fa-plus"></i></span>MongoDB
+            </li>
+            <li>
+              <span><i class="fas fa-plus"></i></span>Microsoft SQL & MySQL
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="column">
+        <img
+          src="./images/headshot_small.jpeg"
+          class="headshot"
+        />
+        <div class="resume-button">
+          <a
+            href="./files/Torres Torres, Mariana_RESUME.pdf"
+            download="MarianaTorresTorresResume"
+          >
+            <button>
+              GET MY RESUME
+            </button>
+          </a>
+        </div>
+      </div>
+    </div>
+    <div id="experience">
+      <h1>
+        Work Experience
+      </h1>
+      <h2>
+        Here is where my work experience is going to be. It's not much, but it's
+        honest ~ work ~
+      </h2>
+    </div>
+    <div id="projects">
+      <h1>
+        Projects
+      </h1>
+      <div>
+        <div class="column">
+          <h3>
+            MEMBERSHPEUF.COM
```

Comment:
Just double checking, but this URL didn't work for me? Totally fine to keep it in, just wanted to give you a heads up!

Reply:
That's because it's not a link yet! I'll make it into a link tomorrow :)